vms/platformvm/state/l1_validator.go

```diff
@@ -23,43 +23,43 @@ import (
 )
 
 var (
-	_ btree.LessFunc[SubnetOnlyValidator] = SubnetOnlyValidator.Less
-	_ utils.Sortable[SubnetOnlyValidator] = SubnetOnlyValidator{}
+	_ btree.LessFunc[L1Validator] = L1Validator.Less
+	_ utils.Sortable[L1Validator] = L1Validator{}
 
-	ErrMutatedSubnetOnlyValidator     = errors.New("subnet-only validator contains mutated constant fields")
-	ErrConflictingSubnetOnlyValidator = errors.New("subnet-only validator contains conflicting subnetID + nodeID pair")
-	ErrDuplicateSubnetOnlyValidator   = errors.New("subnet-only validator contains duplicate subnetID + nodeID pair")
+	ErrMutatedL1Validator     = errors.New("subnet-only validator contains mutated constant fields")
+	ErrConflictingL1Validator = errors.New("subnet-only validator contains conflicting subnetID + nodeID pair")
+	ErrDuplicateL1Validator   = errors.New("subnet-only validator contains duplicate subnetID + nodeID pair")
 )
 
-type SubnetOnlyValidators interface {
-	// GetActiveSubnetOnlyValidatorsIterator returns an iterator of all the
+type L1Validators interface {
+	// GetActiveL1ValidatorsIterator returns an iterator of all the
 	// active subnet-only validators in increasing order of EndAccumulatedFee.
```

Comment:
```suggestion
	// active l1 validators in increasing order of EndAccumulatedFee.
```
many comments in this file still mention subnet-only validators